### PR TITLE
used opacity to make the switching smooth

### PR DESCRIPTION
**Pull request by HussainKhanSherwani**
# Pull Request Template

Before submitting this PR, please make sure you have completed the following checklist. This helps ensure that our codebase remains high quality and that our features work well across all supported devices and themes.

### Code Review
- [x] I have carefully reviewed my code and believe it is final from my side.
- [x] I have ensured there is no hardcoded color in my changes. Colors should be defined in theme files and reused.

### Theme Compatibility
- [x] I have tested the new feature or updates in both light and dark themes.
- [x] I confirm that the visual aspects of the feature are compatible and appealing in both themes.

### Device Compatibility
- [x] I have tested my changes on mobile screen sizes.
- [x] I have tested my changes on tablet screen sizes.
- [x] I have tested my changes on laptop screen sizes.

### Functional Testing
- [x] I have thoroughly tested the functionality of my changes.
- [x] I am confident that my changes do not negatively impact existing features.

### Testing Documentation
Please provide a brief description of the tests you have performed to verify your changes. Include any relevant scenarios or edge cases.

- Example test 1: Tested feature X by uploading the image
- Example test 2: Tested feature X by doing these things.

# Demo Video
Please provide a demo video of your changes. This helps reviewers understand the changes and verify the functionality.

You can record the video using tools like Loom(https://www.loom.com/), Clip by ClickUp(https://clickup.com/features/clips), Sendspark(https://www.sendspark.com/), etc.

Include the link to the video below and make sure the video covers
- [x] Explain the task/bug you are solving
- [x] Show the changes in the UI
- [x] Show the changes in the codebase
- [x] Show the changes you tested to make sure it works
- [x] Show the changes in various screen sizes
- [x] Show the changes in both light and dark themes
videoURL:https://www.loom.com/share/fb7f74116c894a1fb917e727412b1a84?sid=7921f2d9-9fd2-4e95-905a-09bbab92e44e

### CODE REVIEW

**Comment by MSamiTariq** [comment]
@HussainKhanSherwani just a small request, can you create a loom comparing how it was before vs now

**Comment by HussainKhanSherwani** [comment]
Fine

**Comment by HussainKhanSherwani** [comment]
> @HussainKhanSherwani just a small request, can you create a loom comparing how it was before vs now

https://www.loom.com/share/8488e1059e42440bbe8ec75448d0959a?sid=27251884-6554-4cd8-85f6-a36a243b0777

**Comment by MSamiTariq** [comment]
> > @HussainKhanSherwani just a small request, can you create a loom comparing how it was before vs now
> 
> https://www.loom.com/share/8488e1059e42440bbe8ec75448d0959a?sid=27251884-6554-4cd8-85f6-a36a243b0777

Thanks!

File: academy-ui/src/components/clickableDemos/View/ClickableDemoModal.tsx [hunk 1]
```diff
@@ -82,7 +92,13 @@ function ClickableDemoModal({ clickableDemoWithSteps, space, onClose }: Clickabl
       iframeArr[i] = document.createElement('iframe');
       iframeArr[i].src = clickableDemoWithSteps.steps[i].url;
       iframeArr[i].width = '100%';
-      iframeArr[i].style.height = i === 0 ? '93vh' : '0vh';
+      iframeArr[i].style.opacity = i === 0 ? '1' : '0';
+      iframeArr[i].style.pointerEvents = i === 0 ? 'auto' : 'none';
+      iframeArr[i].style.transition = 'opacity 0.3s ease-in-out'; // Smooth transition for opacity
```
Comment (MSamiTariq) [inline academy-ui/src/components/clickableDemos/View/ClickableDemoModal.tsx:97]: can we not have the same transition using our previous logic of changing height?

Comment (HussainKhanSherwani) [reply academy-ui/src/components/clickableDemos/View/ClickableDemoModal.tsx:97]: > can we not have the same transition using our previous logic of changing height?

no i looked into it but it was breaking u

**Review by MSamiTariq — APPROVED** [review]
Looks good to me
